### PR TITLE
 category_encoders 1.3.0

### PR DESCRIPTION
**Pull request by Arishamays1**
## ☆ Category_encoders 1.3.0 Update ☆
[Jira Ticket](https://anaconda.atlassian.net/browse/PKG-2511)
[Upstream](https://github.com/scikit-learn-contrib/category_encoders)
# Changes
- Updated version number and `sha256`
- Updated pinnings on dependencies

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,52 +1,43 @@
 {% set name = "category_encoders" %}
-{% set version = "2.6.1" %}
+{% set version = "1.3.0" %}
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:2]: You are trying to update to the version that is lower than we have on defaults. You should open the PR against the branch `master-1.3.0` but not `master`. See docs https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/02_Package_building/01_How_tos/Creating_versioned_feedstocks_from_existing_packages.md

Comment (Arishamays1) [reply recipe/meta.yaml:2]: Thanks Serhii for catching this!

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,52 +1,43 @@
...
+    - scikit-learn >=0.15.0
+    - scipy >=0.9
+    - statsmodels >=0.6.0
 
 test:
-  source_files:
-    - tests
   requires:
     - pip
     - pytest
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
```

**Review by lorepirri — APPROVED** [review]
LGTM! I think `pytest` is not needed anymore because the tests are not run

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,52 +1,42 @@
...
-  build:
-    - patch       # [not win]
-    - m2-patch    # [win]
   host:
     - python
     - pip
     - wheel
     - setuptools
   run:
+    - numpy >=1.8.0
```
Comment (cbouss) [inline recipe/meta.yaml]: from setup.py from pypi archive
```
    install_requires=[
      'numpy>=1.11.1',
      'scikit-learn>=0.17.1',
      'scipy>=0.17.0',
      'statsmodels>=0.6.1',
      'pandas>=0.20.1',
      'patsy>=0.4.1',
    ],
```

**Review by cbouss — APPROVED** [review]
LGTM, provided deps constraints are updated.

**Review by skupr-anaconda — COMMENTED** [review]
It can be approved after addressing Charles's comment

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**